Sources/Shared/UI/Documents/DocumentsViewController.m

```diff
@@ -1068,15 +1077,16 @@ - (void)imagePickerController:(UIImagePickerController *)picker didFinishPicking
                 
                 //release the date formatter because, not needed after that piece of code
                 [dateFormatter release];
-                imageName = [NSString stringWithFormat:@"MobileImage_%@.png", tmp];
+                imageName = [NSString stringWithFormat:@"mobile_image_%@.png", tmp];
```

Comment:
Just a question: why did you change the filename prefix?
Also, this prefix should be defined as a constant in defines.h and everywhere necessary.